tests/common.py

```diff
@@ -0,0 +1,124 @@
+import logging
+import json
+import os
+
+import pyarrow as pa
+from pyarrow.lib import tobytes
+from pyarrow.lib import Table
+
+# create and configure main logger
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+
+# create console handler with a higher log level
+LOG_HANDLER = logging.StreamHandler()
+LOG_HANDLER.setLevel(logging.DEBUG)
+
+# create formatter and add it to the handler
+FORMATTER = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+LOG_HANDLER.setFormatter(FORMATTER)
+
+# add the handler to the logger
+logging.getLogger('').addHandler(LOG_HANDLER)
+
+
+def get_full_path(file_names: list[str]) -> list[str]:
+    realpath_directory = os.path.realpath(
+        os.path.join(os.getcwd(), os.path.dirname(__file__)))
+    data_dir = os.path.join(realpath_directory, 'data/tpch_parquet')
+    full_paths_list = (
+        [os.path.join(data_dir, dataset) for dataset in file_names]
+    )
+
+    return full_paths_list
+
+
+def get_substrait_plan(filename: str) -> str:
+    realpath_directory = os.path.realpath(
+        os.path.join(os.getcwd(), os.path.dirname(__file__)))
+    plan_dir = os.path.join(realpath_directory, 'integration/queries/tpch_substrait_plans')
+    plan_path = os.path.join(plan_dir, filename)
+
+    f = open(plan_path, 'r')
+    plan = json.loads(f.read())
+
+    return json.dumps(plan)
```

Comment:
```suggestion
    with open(plan_path, "r") as f:
        return f.read()
```

I may be misunderstanding what's happening here -- but right now we load from a string into a json object, then dump it out as a string.  If the idea is to validate the JSON, that should be called out in a separate function.  Otherwise, this could just be returning the contents of the file as a string.